runtime/compiler/env/j9methodServer.cpp

```diff
@@ -970,11 +964,11 @@ TR_ResolvedJ9JITServerMethod::getROMString(int32_t &len, void *basePtr, std::ini
    for (size_t offset : offsets)
       {
       ptr += offset;
-      if (!inROMClass(ptr))
+      if (!JITServerHelpers::isAddressInROMClass(ptr, romClassPtr()))
```

Comment:
Since `romClassPtr()` is called in a loop, could we avoid the virtual call by caling it once before the loop and useing the returned value inside?